### ISSUE TITLE
Tomorrow valid flag

### ISSUE THREAD

**Issue by ollan** [custom-components/nordpool#82]
It seems that the "Tomorrow valid" flag does not work properly. I use SE3 region and also this morning the flag showed "true" value.
One proposal would be to look through all values in tomorrow array. If all 24 values are filled than set "tomorrow valid" to "true".

**Comment by Hellowlol** [comment]
I have never seen this before. Usually things will blow up and create a exception. Disk you see anything in the log?

**Comment by ollan** [comment]
No log occured.
I will try to get some data tomorrow morning when "tomorrow_valid" should not be "true".

**Comment by Repsionu** [comment]
I can confirm erratic behavior of "tomorrow_valid" after DST change :-(
After midnight it changes to "True" as data here (Estonia) has data for 1.00 hour...
Setting it manually to "False" hides my conditional card but after todays update (ca. 14.00) it is not changed.
Best, JR

**Comment by ThoStaKa** [comment]
Same issue to me as well with latest version (0.0.4). 
Flag is set to true without data for tomorrow (time is 08:50)
No errors in log, tried reloading the integration and several reboots

**Comment by nord-** [comment]
When values for tomorrow all contains null, the `tomorrow_valid`-flag should be `false`.

Example of current behaviour:
```
tomorrow_valid: true
tomorrow:
  - null
raw_tomorrow:
  - start: '2022-01-28T00:00:00+01:00'
    end: '2022-01-28T01:00:00+01:00'
    value: null
```

**Comment by ThoStaKa** [comment]
Right now, with HA 2022.2 I have a new day with wrong flag for tomorrow. It still seems very intermittent and worked well yesterday.

`tomorrow_valid: true
today: 15.206, 15.057, 15.169, 15.431, 15.344, 15.169, 15.269, 15.618, 16.004, 16.079, 16.141, 16.179, 15.991, 15.941, 16.054, 15.979, 15.456, 15.256, 15.07, 14.684, 14.248, 15.53, 15.356, 15.131
tomorrow: , , , , , , , , , , , , , , , , , , , , , , , `

**Comment by UriasPost** [comment]
I am also seeing erratic behaviour of this field. It might be dependent upon the time of day of the request and the "real" availability of tomorrow's data? Could it be that tomorrow_valid is set based on time, not actually available data? When testing some minutes ago (at 12:50 CET) I get true for tomorrow_valid with only null values. 15 minutes later (at 13:05 CET), the values are present (not null).

**Comment by sepahewe** [comment]
I also see this behavior some days, but only intermittently. 

My workaround is to check if there exists data for .tomorrow as a safety:
```
{% if state_attr('sensor.nordpool_kwh_se3_sek_3_10_025', 'tomorrow_valid') == true and state_attr('sensor.nordpool_k
wh_se3_sek_3_10_025', 'tomorrow')|length > 24  -%}
```

**Comment by Hellowlol** [comment]
Fixed in 0.0.8
